scalecube-configuration/src/main/java/io/scalecube/configuration/tokens/OrganizationServiceKeyProvider.java

```diff
@@ -38,6 +39,7 @@ public OrganizationServiceKeyProvider(OrganizationService organizationService) {
                 keyId ->
                     organizationService
                         .getPublicKey(new GetPublicKeyRequest(keyId))
+                        .publishOn(Schedulers.parallel())
```

Comment:
Regarding this. Let's wait Dima's  PR to scalecube-security (with asynchronous `JwtKeyResolver`)